superset-frontend/plugins/plugin-chart-echarts/test/Timeseries/index.test.ts

```diff
@@ -0,0 +1,28 @@
+/**
++ * Licensed to the Apache Software Foundation (ASF) under one
++ * or more contributor license agreements.  See the NOTICE file
++ * distributed with this work for additional information
++ * regarding copyright ownership.  The ASF licenses this file
++ * to you under the Apache License, Version 2.0 (the
++ * "License"); you may not use this file except in compliance
++ * with the License.  You may obtain a copy of the License at
++ *
++ *   http://www.apache.org/licenses/LICENSE-2.0
++ *
++ * Unless required by applicable law or agreed to in writing,
++ * software distributed under the License is distributed on an
++ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
++ * KIND, either express or implied.  See the License for the
++ * specific language governing permissions and limitations
++ * under the License.
++ */
+import { EchartsTimeseriesChartPlugin } from '@superset-ui/plugin-chart-echarts';
+
+describe('@superset-ui/plugin-chart-echarts', () => {
+  it('exists', () => {
+    const chart = new EchartsTimeseriesChartPlugin().configure({
+      key: 'echarts_timeseries',
+    });
+    expect(chart.metadata.parseMethod).toEqual('json');
+  });
+});
```

Comment:
Looking at the test suite, maybe we could simplify and make this more comprehensive by moving this test into `test/index.test.ts` and have a section there that ensures `parseMethod` is correctly defined for all plugins.

Bycatch: I also noticed that the `exists` test there doesn't cover all plugins. While you're at it, can you add the missing plugins, too?